workflows/export/asset_export-vx.go

```diff
@@ -88,38 +104,156 @@ func AssetExportVX(ctx workflow.Context, params AssetExportParams) (*AssetExport
 
 	var mergeResult MergeExportDataResult
 	err = workflow.ExecuteChildWorkflow(ctx, MergeExportData, MergeExportDataParams{
-		ExportData: data,
-		TempPath:   tempFolder,
-		OutputPath: outputFolder,
+		ExportData:   data,
+		TempDir:      tempDir,
+		SubtitlesDir: outputDir,
 	}).Get(ctx, &mergeResult)
 	if err != nil {
 		return nil, err
 	}
 
+	// Destination branching:  VOD, playout, bmm, etc.
+	var resultFutures []workflow.Future
+	for _, dest := range params.Destinations {
+		destination := AssetExportDestinations.Parse(dest)
+		if destination == nil {
+			return nil, fmt.Errorf("invalid destination: %s", dest)
```

Comment:
Should this be done earlier in the flow? I.e. no need to process any data if the destinations are invalid...